.github/workflows/wheels.yml

```diff
@@ -48,7 +58,7 @@ jobs:
       # https://github.com/actions/upload-artifact/issues/478
       - uses: actions/download-artifact@v3
         with:
-          name: "ort-results-s3connectorforpytorch--"
+          name: ${{ ARTIFACT_NAME }}
```

Comment:
Why can't we use hardcoded name here? For what reason we want general solution here?

Reply:
In the private repository, we were wanting to get all of the workflows working, including the wheel generation workflow.

Reply:
ORT uses the repository name to determine what the uploaded artefact will be called, so hardcoding it doesn't work if we want it working between multiple repositories